components/playback/controls/player/AddCustomer.vue

```diff
@@ -6,16 +6,54 @@
     </span>
     <div id="uuid" style="top: 40px; left: 50px; position: absolute">
       <InputWidget
-        :title_label="label_uuid"
-        :placeholder="keyplaceholder_encode"
-        :spellcheck="spellchecking"
+        :title_label="'Enter Alphanumeric ID'"
+        :placeholder="'2VSckkBYr2An3dqHEyfRRE'"
+        :spellcheck="false"
         :invalid_text="error_text_uuid"
-        :input_width="entry_width"
-        :disabled="disallow_entry"
+        :input_width="400"
+        :disabled="false"
         :dom_id_suffix="'alphanumeric-id'"
         @update:value="on_update_uuid($event)"
       ></InputWidget>
     </div>
+
+    <div id="apikey" style="top: 140px; left: 50px; position: absolute">
+      <InputWidget
+        :title_label="'Enter API Key(Optional)'"
+        :placeholder="'ba86b8f0-6fdf-4944-87a0-8a491a19490e'"
+        :spellcheck="false"
+        :invalid_text="error_text_api"
+        :input_width="400"
+        :disabled="false"
+        :dom_id_suffix="'apikey-id'"
+        @update:value="on_update_api($event)"
+      ></InputWidget>
+    </div>
+    <div id="nickname" style="top: 240px; left: 50px; position: absolute">
+      <InputWidget
+        :title_label="'Enter ID Nickname'"
+        :placeholder="'Curi Bio Main Account'"
+        :spellcheck="false"
+        :invalid_text="error_text_nickname"
+        :input_width="400"
+        :disabled="false"
```

Comment:
the default value for this prop is already `false`, so it's not necessary to specify that here

Reply:
Agreed